package.json

```diff
@@ -4,7 +4,7 @@
   "license": "ISC",
   "description": "Coinbase Platform SDK",
   "repository": "https://github.com/coinbase/coinbase-sdk-nodejs",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
If you run `npm install` does that bump `package-lock.json`?

Reply:
updated